node-admin/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 install(DIRECTORY DESTINATION logs/vespa/node-admin)
 install(FILES target/node-admin-jar-with-dependencies.jar DESTINATION conf/node-admin-app/components)
+install_symlink(lib/jars/application-model-jar-with-dependencies.jar conf/node-admin-app/components/application-model.jar)
```

Comment:
Why not preserve the '-jar-with-dependencies' suffix?

Reply:
For readability.  The "-jar-with-dependencies" part is ugly and long.  They are typically removed for config server, but preserved here for node-admin. I could just as easily have included the part for consistency - let me know if you prefer that.

Reply:
Agree that it's ugly, we should have used just "-fat' or "-deploy" instead. But for now, let's be consistent. It's also convenient to see from the file names which jars are fat and not.

Reply:
Done